skimage/transform/tests/test_polar.py

```diff
@@ -0,0 +1,60 @@
+import numpy as np
+from skimage.draw import circle
+from skimage.transform import cart2pol
+from numpy.testing import assert_equal
```

Comment:
I usually opt for `assert_allclose` for unit tests, as different / faster implementations in the future may yield the "same" results within floating point error. `assert_allclose` allows these to pass, while `assert_equal` is overly restrictive.